src/main.py

```diff
@@ -738,6 +759,15 @@ def main() -> None:
         action="store_true",
         help="Debug mode, only compiles and scores the base for debugging issues",
     )
+    parser.add_argument(
+        "--speed",
+        dest="speed",
+        type=int,
+        help="Speed% to run at to reduce resources. Default 100",
+        choices=range(0,101),
+        metavar="[0-100]",
```

Comment:
```suggestion
        choices=range(1,101),
        metavar="[1-100]",
```
to avoid division by zero